src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,132 @@
 package core.basesyntax;
 
 public class SalaryInfo {
-    public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+    public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo)
+            throws DataGettingException {
```

Comment:
The method `getSalaryInfo` should use `LocalDate` for date handling instead of manual parsing. This will simplify the date comparison logic and improve accuracy.